pay-api/src/pay_api/services/cfs_service.py

```diff
@@ -367,6 +368,16 @@ def create_account_invoice(cls, transaction_number: str, line_items: List[Paymen
                                            invoice_payload)
         return invoice_response.json()
 
+    @classmethod
+    def _build_reversal_comment(cls, operation: ReverseOperation):
+        """Build the comment for the reversal."""
+        return {
```

Comment:
Put this in to make it a bit more robust. Going to confirm with CAS. 
Top two exist already. 

Correction, the idea was to increment `cas_version_suffix`, and append it to a routing slip number if it's > 1

EX.

`234054347`, `cas_version_suffix` incremented becomes `234054347R2`... `234054347R3` etc..